orbit/pkg/update/update.go

```diff
@@ -152,11 +151,7 @@ func NewDisabled(opt Options) *Updater {
 // UpdateMetadata downloads and verifies remote repository metadata.
 func (u *Updater) UpdateMetadata() error {
 	if _, err := u.client.Update(); err != nil {
-		// An error is returned if we are already up-to-date. We can ignore that
-		// error.
-		if !client.IsLatestSnapshot(ctxerr.Cause(err)) {
```

Comment:
Looks like this check is no longer necessary (see https://github.com/theupdateframework/go-tuf/commit/ed6788e710fc3093a7ecc2d078bf734c0f200d8d#diff-e96ae16d489c5f2c4a06f8bbf477dd55ce1aef9d6f69ad5ce2731fda30416899L34) and the function has been removed (see https://github.com/theupdateframework/go-tuf/commit/ed6788e710fc3093a7ecc2d078bf734c0f200d8d#diff-f12f1bb4e14e9b4f35a62b18aacb25da8cfa219b7a8f1cfeabf6715be2430723L73-L84)